README.md

```diff
@@ -115,6 +115,7 @@ Sources:
 * rpm
 * deb
 * node packages (npm)
+* pacman (`pkg.tar.xz`, or ArchLinux) packages
```

Comment:
Other distros use Pacman format packages (Frugalware for example) and there are also other compression systems allowed. I know it seems weird for Arch users to not have everything up to date but for various reasons relating to that, a flag to allow selecting of lzma/xz/gz/bz2 would be a nice addition.